### PR TITLE
refactor: pricing component

### PR DESCRIPTION
**Pull request by Prerna290**
## Type
- ( ) Fix
- (X) Refactor
- ( ) Style
- ( ) Docs
- ( ) Feat
- ( ) Hotfix

## Screenshots
image
image
image

### CODE REVIEW

File: apps/commudle-admin/src/app/feature-modules/pricing/components/pricing/pricing.component.scss [hunk 1]
```diff
@@ -198,6 +198,31 @@ p {
     }
   }
 
+  .section {
+    @apply com-px-6 com-my-12 com-mx-72px;
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/pricing/components/pricing/pricing.component.scss]: is this mx needed?

File: apps/commudle-admin/src/app/feature-modules/pricing/components/pricing/pricing.component.scss [hunk 2]
```diff
@@ -198,6 +198,31 @@ p {
...
+
+    div {
+      @apply com-flex com-flex-row com-justify-between com-m-0 com-p-0 com-gap-x-72;
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/pricing/components/pricing/pricing.component.scss]: why are we giving 72 gap here (72 = 288 px)?

File: apps/commudle-admin/src/app/feature-modules/pricing/components/pricing/pricing.component.scss [hunk 3]
```diff
@@ -198,6 +198,31 @@ p {
...
+
+      .content {
+        @apply com-mt-12 com-mx-4 com-flex com-flex-col;
+
+        h1 {
+          @apply com-mb-4 com-p-0 com-text-4xl com-font-extrabold com-mt-0;
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/pricing/components/pricing/pricing.component.scss]: h1 already has a 36px font size, then why we are giving font size here?

**Review by ArshdeepGrover — CHANGES_REQUESTED** [review]

File: apps/commudle-admin/src/app/feature-modules/pricing/components/pricing/pricing.component.scss (hunk 3 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/pricing/components/pricing/pricing.component.scss]: Solved

**Pull request merged**